jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridViewImpl.java

```diff
@@ -570,280 +570,150 @@ private void initGroupTabFilter( GridGlobalPreferences preferences,
                                      final String key,
                                      String tabName,
                                      String tabDesc,
-                                     List<String> states,
-                                     String role ) {
+                                     List<String> states ) {
         FilterSettingsBuilderHelper builder = FilterSettingsBuilderHelper.init();
         builder.initBuilder();
 
         builder.dataset(HUMAN_TASKS_WITH_USER_DATASET);
-        List<Comparable> names = new ArrayList<Comparable>();
-
-        for ( String s : states ) {
-            names.add( s );
-        }
+        List<Comparable> names = asComparables(states);
         builder.filter( COLUMN_STATUS, equalsTo( COLUMN_STATUS, names ) );
```

Comment:
you probably don't need the `asComparables` method, if you do `new ArrayList<>(states)` in here.